src/transport_picoquic.cpp

```diff
@@ -1599,3 +1607,32 @@ void PicoQuicTransport::close_stream(const ConnectionContext& conn_ctx, DataCont
 
     data_ctx->current_stream_id = 0;
 }
+
+void PicoQuicTransport::mark_stream_active(const TransportConnId conn_id, const DataContextId data_ctx_id) {
+    const auto conn_it = conn_context.find(conn_id);
+    if (conn_it == conn_context.end()) {
+        return;
+    }
+
+    const auto data_ctx_it = conn_it->second.active_data_contexts.find(data_ctx_id);
+    if (data_ctx_it == conn_it->second.active_data_contexts.end()) {
+        return;
+    }
+    
+    picoquic_mark_active_stream(conn_it->second.pq_cnx,
+                                data_ctx_it->second.current_stream_id,1,
```

Comment:
Not a big deal, but that `,1` I almost missed there. I'd suggest putting that on a separate line or at least adding a space.

Reply:
Updated